.circle/deployment

```diff
@@ -21,6 +21,20 @@ fi
 # TODO: remove the install_gh call once circleci config is updated in all packs
 type gh &>/dev/null || ~/ci/.circle/install_gh
 
+# GH_TOKEN is used by gh which is used in functions.sh
+export GH_TOKEN=${MACHINE_PASSWORD}
+source ~/ci/tools/functions.sh
+
+DEFAULT_BRANCH=$(_gh_default_branch)
+# Give a good error message after the first use of github API in this script.
+if $? != 0; then
```

Comment:
Does this need to be in a test function?

```suggestion
if [[ $? != 0 ]]; then
```

Reply:
Good catch. Thanks!